cypress/fixtures/registration/valid-user.json

```diff
@@ -1,8 +1,17 @@
 {
-  "first_name": "Edward",
-  "last_name": "Withers",
-  "email": "test@test.com",
-  "password": "test12",
-  "biography": "a long bio",
-  "github_url": "https://github.com/dearshrewdwit"
+  "status": "success",
+  "data": {
+    "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjUsImlhdCI6MTY0OTQxMzk0OSwiZXhwIjoxNjQ5NTAwMzQ5fQ.b37lSRtpFWJ9kqUYAc6PUIP28JXjAYtBN_GpU5TcEuc",
```

Comment:
why did you modify the valid-user.json? the updated json is the response you're getting from the server, which includes the login access token

Reply:
We checked with Lewis but It's because we had a status error and the object didn't contain status before.